pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/DDPRequestUtilTest.java

```diff
@@ -23,13 +25,15 @@ public static void first() throws Exception {
         setupDB(true);
         startMockServer();
         setupUtils();
+        DBTestUtil.makeTestDDPInstanceActive(true);
     }
```

Comment:
Using that new method. A lot of the tests assume the test instance is active, but somewhere in some test we were making inactive. So every test that counts on it being active will have this line.